README.md

```diff
@@ -47,35 +45,47 @@ Workers are built in their own bin directories:
 * Client: `OtherWorkers/Interactive/client/bin`
 
 ## Cleaning the project
-Run `./clean.sh` to delete all build files, including worker binaries and any intermediate files generated during the build process.
+Run `./clean.sh` to delete all build files, including worker binaries and any intermediate files generated during the build process. Alternatively, run `git clean -xdf`.
```

Comment:
What does running `git clean -xdf` do differently? Is this something we should explain here?

Reply:
This is a common git command that you can google when in doubt.

Reply:
what about adding a sentence explaining what `git clean -xdf` does here? if users created new files (e.g. new types of workers) and follow the instruction without knowing what it actually does, they might be deleting useful files too and these files are gone because `git clean -xdf` cleans all untracked files, not only all build files.